### PR TITLE
chore: changelog 1.20.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore: changelog 1.20.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+- False-positive error on let-defined variable. See [#993](https://github.com/redhat-developer/quarkus-ls/issues/993).
+- Fix the qute.jdt tests. See [#1019](https://github.com/redhat-developer/quarkus-ls/pull/1019).
+- Support TemplateExtension matchNames. See [#1013](https://github.com/redhat-developer/quarkus-ls/issues/1013).
+- Qute validation fails on `{config:property}` if property contains is a string. See [#933](https://github.com/redhat-developer/quarkus-ls/issues/933).
+- Cannot publish to OpenVSX due to out of date NodeJS. See [vscode-microprofile#303](https://github.com/redhat-developer/vscode-microprofile/issues/303).
+
+### Build
+
+- Bump webpack from 5.94.0 to 5.95.0. See [vscode-microprofile#296](https://github.com/redhat-developer/vscode-microprofile/pull/296).
+- Update vscode-redhat-telemetry to 0.9.0.. See [vscode-microprofile#293](https://github.com/redhat-developer/vscode-microprofile/pull/293).
```
Comment (datho7561) [inline CHANGELOG.md]: I don't think the end users will care about these, so I think we should leave them out

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+- False-positive error with section name which contains number. See [#1022](https://github.com/redhat-developer/quarkus-ls/pull/1022).
+- False-positive error on let-defined variable. See [#993](https://github.com/redhat-developer/quarkus-ls/issues/993).
+- Fix the qute.jdt tests. See [#1019](https://github.com/redhat-developer/quarkus-ls/pull/1019).
+- Support TemplateExtension matchNames. See [#1013](https://github.com/redhat-developer/quarkus-ls/issues/1013).
+- Qute validation fails on `{config:property}` if property contains is a string. See [#933](https://github.com/redhat-developer/quarkus-ls/issues/933).
+- Cannot publish to OpenVSX due to out of date NodeJS. See [vscode-microprofile#303](https://github.com/redhat-developer/vscode-microprofile/issues/303).
+
+### Build
+
+- Bump webpack from 5.94.0 to 5.95.0. See [vscode-microprofile#296](https://github.com/redhat-developer/vscode-microprofile/pull/296).
```
Comment (datho7561) [inline CHANGELOG.md]: Also this one, I don't think we should show anything from the build section

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (datho7561) [reply CHANGELOG.md]: I made this change myself

**Review by datho7561 — APPROVED** [review]

**Pull request merged**